.github/workflows/vuepress.yml

```diff
@@ -0,0 +1,31 @@
+name: Vuepress
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Vuepress deploy
+        uses: jenkey2011/vuepress-deploy@dev
+        env:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
There will need to be a secret added that contains a value for an access token for deployment.